deploy/base/model.yaml

```diff
@@ -1,2 +1,4 @@
 small: embeddings_small_2023_05_24_23_17_03.csv
 medium: embeddings_medium_2023_05_24_22_15_24.csv
+finetuned_alternatives_model: ft_alt_all-MiniLM-L6-v2_v20230623171244
```

Comment:
Why do we need the base in the yaml since its not consumed by the nl-server (and the full version history is part of the final model name)?

Reply:
Hmm. I thought it would be good to have it but your suggestion makes sense. Removed.